### PR TITLE
[C-2178] Complete w/ tiktok qa fixes

### PR DESCRIPTION
**Pull request by sliptype**
### Description

* Fix verified icon styles
* Disable handle inputs on mobile when verified
* Update feature flag
* Show disabled input state on web

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx [hunk 1]
```diff
@@ -95,18 +108,21 @@ const EditProfileForm = (props: FormikProps<ProfileValues>) => {
         <FormTextInput name='bio' label='Bio' multiline maxLength={256} />
         <FormTextInput name='location' label='Location' />
         <FormTextInput
+          editable={!isTikTokVerified}
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx:111]: awesome additions!

**Review by nicoback2 — APPROVED** [review]

File: packages/mobile/src/components/core/FormTextInput.tsx [hunk 2]
```diff
@@ -76,6 +79,7 @@ export const FormTextInput = ({
     name,
     validate: required ? validateRequired : undefined
   })
+  const disabled = other.editable === false
```
Comment (dylanjeffers) [inline packages/mobile/src/components/core/FormTextInput.tsx:82]: do we want to pass editable down with other, or destructure it just for this component?

File: packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx:111]: i personally prefer "readOnly" as the prop name, there is some precedent for this name, heres MUI: 
image

**Review by dylanjeffers — CHANGES_REQUESTED** [review]
great changes, i do feel like readOnly is better, but curious what you think

File: packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx:111]: oh jk, i see editable as a prop for text-input, okay ignore all this and the one below

**Review by dylanjeffers — APPROVED** [review]
carry on :)

File: packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx (hunk 1 above)
Comment (sliptype) [reply packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx:111]: readOnly is also a prop for TextInput though, I could change it to that

Comment (dylanjeffers) [reply packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx:111]: no worries lets stay with react-native api

Comment (sliptype) [reply packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx:111]: It is! https://reactnative.dev/docs/textinput#readonly

Comment (dylanjeffers) [reply packages/mobile/src/screens/edit-profile-screen/EditProfileScreen.tsx:111]: oh wait, im so confused lmao

**Pull request merged**